core/src/retransmit_stage.rs

```diff
@@ -352,7 +356,7 @@ pub fn retransmitter(
     bank_forks: Arc<RwLock<BankForks>>,
     leader_schedule_cache: Arc<LeaderScheduleCache>,
     cluster_info: Arc<ClusterInfo>,
-    shreds_receiver: Receiver<Vec<Shred>>,
+    shreds_receiver: Receiver<Vec</*shred:*/ Vec<u8>>>,
```

Comment:
nit: `Receiver<Vec<ShredPayload>>` or `ShredReceiver` if you want to go that route.

Reply:
I will revisit this once follow up changes are in and this code is stable, thanks
https://github.com/solana-labs/solana/pull/26230#discussion_r910431525